src/probes/index.js

```diff
@@ -15,9 +15,11 @@ import isUnaryExpression from "./isUnaryExpression.js";
 import isWeakCrypto from "./isWeakCrypto.js";
 import isClassDeclaration from "./isClassDeclaration.js";
 import isMethodDefinition from "./isMethodDefinition.js";
+import isUnsafeImport from "./isUnsafeImport.js";
```

Comment:
I would prefer isUnsafeEvalRequire

Reply:
Make more sense totally agree, and it's fixed in latest commit